base_geolocalize_openstreetmap/models/geolocalize_openstreetmap.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+import logging
+import requests
+
+from odoo import api, exceptions, models, _
+
+_logger = logging.getLogger(__name__)
+
+
+class GeoengineGeolocalizeOpenStreetmap(models.AbstractModel):
+
+    _name = 'geoengine.geolocalize.openstreetmap'
+
+    @api.model
+    def _geocode_address(
+            self, street, zip_code, city, state, country_name,
+            country_codes):
+        """Get the latitude and longitude by requesting "mapquestapi"
+        see http://open.mapquestapi.com/geocoding/
+        """
+        url = 'http://nominatim.openstreetmap.org/search'
```

Comment:
as it is an openstreetmap class then could it make sens to externalize this one into a class attribute `_url` ?? (in order to reuser request into other call?)

Reply:
Yes, you are right. I also thought about using a config parameter but I think it doesn't make sense as this URL will maybe never change.